src/palette/internal/palettecreator.cpp

```diff
@@ -1327,7 +1346,7 @@ PalettePtr PaletteCreator::newLinesPalette(bool defaultPalette)
     staffTextLine->setEndHookType(HookType::HOOK_90);
     sp->appendElement(staffTextLine, QT_TRANSLATE_NOOP("palette", "Staff text line"));
 
-    auto systemTextLine = makeElement<TextLine>(gpaletteScore);
+    auto systemTextLine = makeElement<TextLine>(gpaletteScore, true);
     systemTextLine->setSystemFlag(true);
```

Comment:
Really the last comment: now this `setSystemFlag` line is redundant because that already happens inside the TextLine constructor

Reply:
Fixed it myself :)